index.php

```diff
@@ -123,7 +132,15 @@
         $genratecookie = has_capability('local/signin:manage', $context);
 
         if (!empty($values->rememberme) && ($genratecookie)) {
-            setcookie($cookiename, rc4encrypt($values->username), time() + (DAYSECS * 60), $CFG->sessioncookiepath,$CFG->sessioncookiedomain, is_moodle_cookie_secure(), $CFG->cookiehttponly);
+            setcookie($cookiename, rc4encrypt($values->username), time() + (DAYSECS * 60), $CFG->sessioncookiepath,
+                $CFG->sessioncookiedomain, is_moodle_cookie_secure(), $CFG->cookiehttponly);
+        }
+
+        if (isset($values->remembermedays)) {
+            global $USER;
+            $userDetailsArray = $USER->id;
```

Comment:
We shouldn't need to use the variable type in the name e.g. $userDetails should be fine here

Reply:
@hannahlillis 
Changes has been made. Please check & Review.

Reply:
Is this actually an array or just an id? If it's just an int, it can just be called $userId, I think $userIdDetails is a little confusing

Reply:
Yupp changed to $userId.